product/roundhouse.databases.sqlserver2000/SqlServerDatabase.cs

```diff
@@ -93,15 +93,19 @@ public override void run_database_specific_tasks()
 
         public override string create_database_script()
         {
+            //Using global variable @@VERSION to get the information about current MS SQL Server instance.
             return string.Format(
                 @"
                         DECLARE @Created bit
                         SET @Created = 0
-                        IF NOT EXISTS(SELECT * FROM sys.databases WHERE [name] = '{0}') 
-                         BEGIN 
-                            CREATE DATABASE [{0}] 
-                            SET @Created = 1
-                         END
+                        IF(PATINDEX('%Microsoft SQL Server 2000%',@@VERSION) > 0)

```

Comment:
What is this call? I see the note above but I still don't know what you are doing here. And what happens when the number is 0? I don't see a call to error?